### PR TITLE
feat(blog): blogpost about gov redeployment

### PR DESCRIPTION
**Pull request by clemsos**
# Description

A blogpost about governor contract migration.

NB: the link to the active DAO proposal need to be added to the blog post once it has been submitted

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/blog/governor-migration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
```
Comment (julien51) [inline unlock-protocol-com/blog/governor-migration.md:2]:
```suggestion
title: Proposed Upgrade of the Unlock DAO Governor Contract 
```

File: unlock-protocol-com/blog/governor-migration.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## **The king is dead, long live the king!**
+
+For the DAO, a change in governor requires two actions: canonize the new one and retire the old one. The vehicle for that process will be [a proposal](https://www.tally.xyz/gov/unlock/proposal/36208249270120864100503453462134662510103434369621143761091232235939585571890) containing the instructions to make the change. Once executed, this DAO popemobile will transport the voting system to its new location.
+
+As a user/voter, there won’t be any change in delegations or voting process. The only apparent change is that the previous proposals will no longer appear in the Tally UI or be visible from the new governor contract.
+
+## **What to do now**
+
+- [Delegate your votes](https://unlock-protocol.com/guides/delegation/) (you **must** delegate in order to participate in onchain votes, including self-delegation)
```
Comment (julien51) [inline unlock-protocol-com/blog/governor-migration.md:26]:
```suggestion
-  If you own any amount of UDT, the Unlock protocol's governance tokens, [delegate your votes](https://unlock-protocol.com/guides/delegation/) (you **must** delegate to participate in onchain votes, including self-delegation)
```

File: unlock-protocol-com/blog/governor-migration.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Head to [the proposal](https://www.tally.xyz/gov/unlock/proposal/36208249270120864100503453462134662510103434369621143761091232235939585571890)
+- Cast your vote!
+
+The voting period ends on **xxx July 2023.** At the end of the voting period, and if the proposal to upgrade passes, the proposal will be queued and executed and the change will become effective.
```
Comment (julien51) [inline unlock-protocol-com/blog/governor-migration.md:30]:
```suggestion
The voting period ends on **July 11th 2023**. At the end of the voting period, and if the proposal to upgrade passes, the proposal will be queued and executed and the change will become effective.
```

**Comment by ccarfi** [comment]
Screen Shot 2023-06-28 at 6 22 23 PM

Screen Shot 2023-06-28 at 6 24 05 PM

**Comment by ccarfi** [comment]
@clemsos Title/headline was doubled in the body of the blog post, and there was no subtitle/description in the metadata, so that was blank on the blog post listing page. I'll try to dive in and fix both and rebuild.

File: unlock-protocol-com/blog/governor-migration.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: Proposed Upgrade of the Unlock DAO Governor Contract
```
Comment (ccarfi) [inline unlock-protocol-com/blog/governor-migration.md:2]:
```suggestion
title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
```

File: unlock-protocol-com/blog/governor-migration.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
+publishDate: June 28, 2023
+authorName: Clément Renaud
+image: /images/blog/ballot-box-image.png
+---
+
+# **Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version**
```
Comment (ccarfi) [inline unlock-protocol-com/blog/governor-migration.md:8]:
```suggestion
```

File: unlock-protocol-com/blog/governor-migration.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
+publishDate: June 28, 2023
+authorName: Clément Renaud
+image: /images/blog/ballot-box-image.png
+---
+
```
Comment (ccarfi) [inline unlock-protocol-com/blog/governor-migration.md:7]:
```suggestion
```

File: unlock-protocol-com/blog/governor-migration.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
+publishDate: June 28, 2023
+authorName: Clément Renaud
+image: /images/blog/ballot-box-image.png
```
Comment (ccarfi) [inline unlock-protocol-com/blog/governor-migration.md:5]:
```suggestion
subTitle: The Unlock DAO deployed its governance contracts in 2021 using the then-current version of the Open Zeppelin Governor contracts. To take advantage of recent evolutions in the state-of-the-art, we believe now is the time for the Unlock DAO to redeploy a new version of the Open Zeppelin governor contracts and benefit from recent improvements.
description: The Unlock DAO deployed its governance contracts in 2021 using the then-current version of the Open Zeppelin Governor contracts. To take advantage of recent evolutions in the state-of-the-art, we believe now is the time for the Unlock DAO to redeploy a new version of the Open Zeppelin governor contracts and benefit from recent improvements.
image: /images/blog/ballot-box-image.png
```

**Review by ccarfi — APPROVED** [review]

**Comment by ccarfi** [comment]
@clemsos reviewed and approved

**Pull request merged**